Sources/GRPC/Interceptor/ClientInterceptorPipeline.swift

```diff
@@ -296,6 +298,9 @@ internal final class ClientInterceptorPipeline<Request, Response> {
 
     // Emit the unwrapped error.
     self._onError(unwrappedError)
+
+    // Close the pipeline.
+    self.close()
```

Comment:
Should `self.close()` not set `self._isOpen`.

Reply:
It does -- but we also need to ensure we do it here because we call out on `_onError`